forest/src/cli/chain_cmd.rs

```diff
@@ -48,6 +69,82 @@ impl ChainCommands {
                 let cid: Cid = cid.parse().unwrap();
                 print_rpc_res_pretty(chain_get_block((CidJson(cid),)).await);
             }
+            Self::Export {
+                tipset,
+                recent_stateroots,
+                skip_old_messages,
+                output_path,
+            } => {
+                let chain_finality = Policy::default().chain_finality;
+                let recent_stateroots = match recent_stateroots {
+                    Some(rsrs) => {
```

Comment:
let's use full name, shadowing is fine :)

Reply:
:+1: